tests/test_annotation_stores.py

```diff
@@ -2115,6 +2142,7 @@ def test_invalid_mode_format(store_cls):
 
     @staticmethod
     def test_invalid_mode(store_cls):
+        """Tests invalid mode for AnnotationStore."""
```

Comment:
```suggestion
        """Check unsupported mode raises ValueError."""
```

Reply:
Addressed in this commit https://github.com/TissueImageAnalytics/tiatoolbox/pull/625/commits/d5bc38668bd092f49acc621c6086ff35776bb23f